File: College-Sports-WebApp/ClientApp/src/api/models/category.ts
/* tslint:disable */
/* eslint-disable */
import { Athlete } from '../models/athlete';
import { League } from '../models/league';
import { Team } from '../models/team';
export interface Category {
  athlete?: Athlete;
  athleteId?: number;
  description?: string | null;
  id?: number;
  league?: League;
  leagueId?: number | null;
  sportId?: number;
  team?: Team;
  teamId?: number | null;
  type?: string | null;
  uid?: string | null;
}

File: College-Sports-WebApp/Database/Models/Category.cs
namespace College_Sports_WebApp.Database.Models{ 

    public class Category
    {
        public int id { get; set; }
        public string description { get; set; }
        public string type { get; set; }
        public int sportId { get; set; }
        public int athleteId { get; set; }
        public Athlete athlete { get; set; }
        public string uid { get; set; }
        public int? teamId { get; set; }
        public Team team { get; set; }
        public int? leagueId { get; set; }
        public League league { get; set; }
    }

}